pages/sign-in/kakao/index.page.tsx

```diff
@@ -3,8 +3,16 @@ import { useEffect } from "react";
 
 import { useRouter } from "next/router";
 
-import { instance } from "@/apis/axios";
+import classNames from "classnames/bind";
+import Lottie from "react-lottie-player";
+
+import { basicInstance, instance, setContext } from "@/apis/axios";
 import { PAGE_PATH } from "@/constants/pagePath";
+import loadingLottie from "@/public/images/login-loading.json";
+
+import styles from "./index.module.scss";
```

Comment:
```suggestion
import styles from "./Kakao.module.scss";
```
P4: 저희 컨벤션에 맞게 수정해도 좋을 것 같아요!

Reply:
헉... 컨벤션에 맞지 않는 파일 이름들 다 수정해두겠습니다ㅠㅠ